Runtimes/Core/CMakeLists.txt

```diff
@@ -105,6 +105,7 @@ option(SwiftCore_ENABLE_SHORT_MANGLING_LOOKUPS "Build with fast-path context des
 option(SwiftCore_ENABLE_FILESYSTEM_SUPPORT "Build for systems that have a filesystem" ON)
 option(SwiftCore_ENABLE_OS_TRACE_LAZY_INIT "Use os_trace call to check if lazy init has been completed before making os_signpost calls." ${SwiftCore_HAS_OS_TRACE})
 option(SwiftCore_HAS_DARWIN_LIBMALLOC "Use Darwin malloc features" ${APPLE})
+option(SwiftCore_ENABLE_CONCURRENCY "Enable Concurrency runtime support" YES)
```

Comment:
Could you disable Concurrency in `Runtimes/Core/cmake/caches/Vendors/Apple/apple-common.cmake`?
We would need a bit more testing before enabling this in Apple internal configurations.

Reply:
Sure!